HangmanGame/ViewController.swift

```diff
@@ -8,12 +8,165 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    @IBOutlet weak var lblScore: UILabel!
+    @IBOutlet weak var correctWord: UILabel!
+    @IBOutlet weak var lblWrongGuess: UILabel!
+    @IBOutlet weak var imgHead: UIImageView!
+    @IBOutlet weak var imgBody: UIImageView!
+    @IBOutlet weak var imgLeftArm: UIImageView!
+    @IBOutlet weak var imgRightArm: UIImageView!
+    @IBOutlet weak var imgLeftLeg: UIImageView!
+    @IBOutlet weak var imgRightLeg: UIImageView!
+    @IBOutlet weak var btnGuess: UIButton!
+    @IBOutlet weak var textBox: UITextField!
+    
+    var score = 0 {
+        didSet {
+            lblScore.text = "Score \(score)"
+        }
+    }
+    
+    var wrongLetters = [String]()
+    var questionMark = [String]()
+    var words = ["MOOSEUM","ROADHOG","TRUNK","MILKSHAKE","BABOOM","BULLDOZER","HISSTORY"]
+    var selectedWord = ""
+    var letters = [Any]()
+    var numberOfLetters = 0
+    var wrongTempLetter = ""
+    var rightTempLetter = ""
+    var counter = 0
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        words.shuffle()
+        loadLevel()
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .refresh, target: self, action: #selector(loadLevel))
+        navigationController?.isToolbarHidden = false
+        
+    }
+    
+    private func divideLetters() {
+        selectedWord = words[1]
+        numberOfLetters = selectedWord.count
+        letters = Array(selectedWord)
+        questionMark.removeAll()
+        correctWord.text = ""
+        for index in 0...numberOfLetters-1{
+            questionMark.append("?")
+            correctWord.text! += questionMark[index]
```

Comment:
Why forced unwrap?!

Reply:
in order to add "?" in a loop. I decided to force unwrap. 
To be safer the new version will have one If statement, in this case, will never be nil.